src/ui/starter-select-ui-handler.ts

```diff
@@ -1323,16 +1331,15 @@ export default class StarterSelectUiHandler extends MessageUiHandler {
       if (this.filterMode && this.filterBar.openDropDown) {
         // CANCEL with a filter menu open > close it
         this.filterBar.toggleDropDown(this.filterBarCursor);
-
-        // if there are possible starters go the first one of the list
-        if (numberOfStarters > 0) {
-          this.setFilterMode(false);
-          this.scrollCursor = 0;
-          this.updateScroll();
-          this.setCursor(0);
+        success = true;
+      } else if (this.filterMode && !this.filterBar.getFilter(this.filterBar.getColumn(this.filterBarCursor)).hasDefaultValues()) {
+        if (this.filterBar.getColumn(this.filterBarCursor) === DropDownColumn.CAUGHT) {
+          this.resetCaughtDropdown();
+        } else {
+          this.filterBar.resetSelection(this.filterBarCursor);
         }
+        this.updateStarters;
```

Comment:
```suggestion
        this.updateStarters();
```
Same comment as the other one.